packages/neuron-ui/src/components/ReplaceDuplicateWalletDialog/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useMemo, useState, useCallback } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import Dialog from 'widgets/Dialog'
+import RadioGroup from 'widgets/RadioGroup'
+import { useState as useGlobalState, useDispatch, updateWalletProperty, setCurrentWallet } from 'states'
+import { RoutePath, isSuccessResponse } from 'utils'
+import { replaceWallet } from 'services/remote'
+import styles from './replaceDuplicateWalletDialog.module.scss'
+
+const useReplaceDuplicateWallet = () => {
```

Comment:
`useReplaceDuplicateWallet` this function is only used in `packages/neuron-ui/src/components/WalletWizard/index.tsx`, why not move to there.

Reply:
`importedWallet` is used in three files

Reply:
> useReplaceDuplicateWallet

My mistake, it is `useReplaceDuplicateWallet` that is used in three files, not `importedWallet`.